source/Mlos.Agent.Server/Mlos.Agent.Server.csproj

```diff
@@ -12,6 +12,10 @@
   </PropertyGroup>
   <Import Project="$(BaseDir)\build\Mlos.NetCore.props" />
   <Import Project="$(BaseDir)\build\Mlos.NetCore.SingleFile.props" />
+  <ItemGroup>
+    <PackageReference Include="CommandLineParser" Version="2.8.0" GeneratePathProperty="true" />
```

Comment:
Is GeneratePathProperty = "true" required ?

Reply:
Ah, no.  That was a holdover from a prior version of this where I was following the pattern we had to do for Mlos.Grpc.Binplace.csproj, but I think we don't want to go that route long term.  I will remove it.

Reply:
Sorry, lied again.  That is required for now for the next line: binplacing.  Let's tackle that in a future PR.